vars/gangplank.groovy

```diff
@@ -0,0 +1,87 @@
+// Run gangplank
+def ganplankCmd
+
+def buildArtifact(params = [:]) {
+// Available parameters:
```

Comment:
For consistency with the other groovy commands, let's put the docstring above the functions?